go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/chenzhuoyu/base64x v0.0.0-20221115062448-fe3a3abad311 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
+	github.com/drone/envsubst v1.0.3 // indirect
```

Comment:
This is an indirect dependency

```shell
❯ go mod why github.com/drone/envsubst
# github.com/drone/envsubst
github.com/go-vela/server/api
github.com/go-vela/types/pipeline
github.com/drone/envsubst
```

Going to update this dependency in `types/pipeline` as well

Reply:
PR to update `types/pipeline` https://github.com/go-vela/types/pull/284